composer.json

```diff
@@ -18,8 +18,7 @@
         "symfony/polyfill-ctype": "^1.8"
     },
     "require-dev": {
-        "phpunit/phpunit": "^4.6",
-        "sebastian/version": "^1.0.1"
```

Comment:
I couldn't find this class being used explicitly, but it is a child dependency for phpunit.

I thinks its better to just remove it at this point, as it doesn't serve a purpose.